tests/30rooms/60version_upgrade.pl

```diff
@@ -510,6 +422,9 @@ sub upgrade_room_synced {
                foreach my $k ( keys %STATE_DICT ) {
                   $levels->{events}->{$k} = 80;
                }
+               # Modify tombstone requirements to 50 so we are able to upgrade the room
+               $levels->{events}->{"m.room.tombstone"} = 50;
```

Comment:
Why? Doesn't the creator have 100?

Reply:
Just a couple lines below we modify creator to be PL50.

Reply:
Hmm, ok, but why? If its trying to test that moderators can upgrade rooms by default then that sounds we're making the wrong change here.

Reply:
Originally moderators were allowed to update rooms by default, and that caused a bug where power levels could split brain. Technically someone could put a room back into this state, so it may still be worth testing?

Originally this PR just deleted this test, but rich thought it good to keep it as it's still a possibility.

Would you modify it in another way?

Reply:
Just don't bother changing the creator to a mod? The test name suggests its trying to test that state gets copied over correctly, so testing that they can do so as a mod was probably just a cheeky extra test

Reply:
AH. No I was thinking of `"/upgrade preserves the power level of the upgrading user in old and new rooms"`. This is a separate test. Yes, we can just pull the PL50 change out of here.